tests/sentry/api/test_utils.py

```diff
@@ -75,3 +78,15 @@ def test_relative_date_range_incomplete(self):
 
         with pytest.raises(InvalidParams):
             start, end = get_date_range_from_params({"timeframeStart": "14d"})
+
+
+class SCIMUtilsTest(SCIMTestCase, SCIMEndpoint):
+    def test_fix_log_name(self):
+        request = HttpRequest()
+        request.user = self.create_user(
+            username="scim-internal-integration-681d6e-ad37e179-501c-4639-bc83-9780ca1"
+        )
+
+        request = self.fix_log_name(request)
+
+        assert request.user.username == "SCIM Internal Integration (681d6e)"
```

Comment:
similar to the previous comments - you _could_ make the `681d6e` into a constant which would make it easy to see what it _should_ be matching.

ex
```
suffix = '681d6e'
...
username=f"scim-internal-integration-{suffix}-ad37e179-501c-4639-bc83-9780ca1"
...
assert request.user.username == f"SCIM Internal Integration ({suffix})"
```

BUT - not really necessary and definitely not a blocker. Completely up to you and whatever you feel might be more readable

Reply:
I will add this, thank you!